src/adapters/plugin/plugin.js

```diff
@@ -178,6 +187,25 @@ class Plugin {
     return this.ipcSocket.sendJson(msg);
   }
 
+  start() {
+    // If we need embedded spaces, then consider changing to npm
+    // module called splitargs
+    let args = this.exec.split(' ');
+    this.process = spawn(args[0], args.slice(1));
+
+    this.process.stdout.on('data', data => {
+      console.log(data.toString());
+    });
+    this.process.stderr.on('data', data => {
+      console.log(data.toString());
+    });
+    this.process.on('close', code => {
+      console.log('Plugin:', this.pluginId, 'died, code =', code,
+                  'restarting...');
+      this.start();
```

Comment:
I do think it would be good to add some sort of rate limiting here, like "if this process died X times in the past minute, don't restart". Something of that nature. Maybe just go ahead and file a separate issue for that so we don't forget?